test/sql/aggregate/aggregates/test_bool.test

```diff
@@ -0,0 +1,130 @@
+# name: test/sql/aggregate/aggregates/test_bool.test
+# description: Test Bool operator
+# group: [aggregates]
+
+# Corner Cases
+statement error
+select bool_or(0)
+
+statement error
+select bool_and(0)
+
+query I
+select bool_or(NULL)
```

Comment:
Can you add a test of bool_or and bool_and with a mix of NULL values and actual values?